data/remnant/remnant outfits.txt

```diff
@@ -678,10 +678,10 @@ outfit "Bellows-Class Afterburner"
 	"outfit space" -7
 	"engine capacity" -7
```

Comment:
```suggestion
	"outfit space" -11
	"engine capacity" -11
```

Reply:
We spent a fair bit of time going over possible installation configurations across most of the Remnant ships, especially the new ones, to allow certain combinations of bellows and engines. This breaks all of that.

Reply:
a) What do you mean by we? I don't recall going over installation configurations other than the default ones.
b) What combinations will not be allowed anymore?

Reply:
The Bellows used 11 outfit space when you originally added it. The only engine space value on any Remnant ship that dates to after it was reduced to 7 is the Smew's 46, which is just enough to fit Bellows+Crucible+Crucible with the Bellows at 11 tons.